README.md

```diff
@@ -2,11 +2,11 @@
 
 This repository contains built-in samples of Azure Policies that can be used as reference for creating and assigning policies to your subscriptions and resource groups. For easy search of all built-in  with descriptions, see [Policy samples](https://docs.microsoft.com/azure/governance/policy/samples/) on docs.microsoft.com.
 
-For custom policy samples, check out our Community repo! (https://github.com/Azure/Community-Policy)
+For custom policy samples, [check](https://github.com/Azure/Community-Policy) out our Community repo! 
```

Comment:
Thanks for this but please put the link on the same words as below (our Community repo!).